scripts/test_usb_scan.py

```diff
@@ -120,7 +120,7 @@ def tearDown(self):
     @nottest
     def test_usb_common(self, moc_devices, moc_interfaces, moc_results,
                         path="./scripts/usb-policy.conf"):
-        import usb_scan
```

Comment:
I noticed you created a separated folder for unit test, why not moving all of `test_*` files into it?